### DISCUSSION TITLE
Allow Additional Config Options from JavaCodegen

### DISCUSSION THREAD

**Discussion by dgautier** [quarkiverse/quarkus-openapi-generator#780] [Ideas]
Hi, 

I'd like to use some features provided by the default Java Generator. (which seems quarkus java generator is based on : by https://openapi-generator.tech/docs/generators/java/ ).

But i can't find a way or documentation to explain how to set options for default java generator, for example generateBuilders to true.

Did i miss something or is it not currently possible to pass config options to Java CodeGen ?

Thanks

**Comment by ricardozanini** [comment]
These properties are not added because we only support some of them, just what makes sense for this extension. Usually, people come with their use case, and we consider adding it. 

This extension is not based on the Java generator but uses the library.

What's your use case?

**Reply by dgautier** [reply]
GenerateBuilders would be nice

**Reply by ricardozanini** [reply]
Feel free to send a PR to add it.
